gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -256,11 +241,9 @@ def event_method_mapping(self, event_id):
         else:
             return event_handler_mapping[event_id]
 
-    @nfp_log.patch_method
     def handle_event(self, event):
+        nfp_logging.store_logging_context(**event.context)
```

Comment:
right now framework is not moved to nfp_logging, it needs to be designed how to handle logging completely in framework.